tests/com/rcv/TabulatorTests.java

```diff
@@ -90,19 +91,25 @@ static boolean fileCompare(String path1, String path2) {
 
   // function: runTabulationTest
   // purpose: helper function to support running various tabulation tests
-  // param: configFile name of  configuration file
-  // param: expectedFile name of expected summary results json file
-  static void runTabulationTest(String configFile, String expectedFile) {
+  // param: stem base name of folder containing config file cvr files and expected result files
+  static void runTabulationTest(String stem) {
     // full path to config file
     String configPath =
-        Paths.get(System.getProperty("user.dir"), TEST_ASSET_FOLDER, configFile)
+        Paths.get(System.getProperty("user.dir"),
+            TEST_ASSET_FOLDER, stem,
+            stem + "_config.json")
```

Comment:
I'm not sure, but this doesn't look like what the automatic reformatting would yield.

Reply:
you know... I have the reformat code option checked in intelliJ but it doesn't seem to pick this up :(

Reply:
I find it works more consistently if you run it manually. The crazy part is that running it manually applies slightly different rules vs. what the commit hook does.